plugins/inputs/postgresql_extensible/README.md

```diff
@@ -0,0 +1,59 @@
+# PostgreSQL plugin
+
+This postgresql plugin provides metrics for your postgres database. It has been designed to parse ithe sql queries in the plugin section of your telegraf.conf.
+
+For now only two queries are specified and it's up to you to add more; some per query parameters have been added :
```

Comment:
Are there any queries that you think are particularly useful beyond the two default queries? Might be worthwhile having some examples at the end of the README of some common queries users might want to do?